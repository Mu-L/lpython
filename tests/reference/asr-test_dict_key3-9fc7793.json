{
    "basename": "asr-test_dict_key3-9fc7793",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_dict_key3.py",
    "infile_hash": "9675711d37ed0e58ddd82a53ec580cc21c58a9b94ad598b706fb78f8",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_dict_key3-9fc7793.stderr",
    "stderr_hash": "46117a6d0b07d8cc2c7ee0cc21bc23f3aca5ae70eedc8bfee72d073e",
    "returncode": 2
}