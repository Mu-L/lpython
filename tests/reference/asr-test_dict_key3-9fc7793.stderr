semantic error: Unhashable type: 'set[Allocatable[str]]'
 --> tests/errors/test_dict_key3.py:4:19
  |
4 |     my_dict: dict[set[str], str] = {{1, 2}: "first", {3, 4}: "second"}
  |                   ^^^^^^^^ Mutable type 'set[Allocatable[str]]' cannot become a key in dict. Hint: Use an immutable type for key.
